### PR TITLE
Remove redundant check

### PR DESCRIPTION
**Pull request by cderb**
FindSolution would itself check for the validity of its result so this extra check in Fin isn't necessary.

### CODE REVIEW

**Comment by cderb** [comment]
Fixes for clang format and make check.

File: src/include/conv_fin.hpp [hunk 1]
```diff
@@ -724,15 +724,6 @@ int ConvFin<Tgpu, Tref>::MIOpenPerfEval()
                 res_item["bias"]           = ctx.problem.bias;
                 res_item["kernel_objects"] = kern_objs;
                 res_item["reason"]         = "Success";
-
-                if(s.IsTunable())
```
Comment (bghimireamd) [inline src/include/conv_fin.hpp:729]: Just curious. We do not need this test any more?

Comment (cderb) [reply src/include/conv_fin.hpp:729]: If you look into MIOpen/src/include/miopen/any_solver.hpp: you can see that GetPerfCfgParams calls the same validation functions as is called in TestPerfCfgParams, so no new information is generated here.

Comment (bghimireamd) [reply src/include/conv_fin.hpp:729]: got it!

**Review by bghimireamd — APPROVED** [review]

**Pull request merged**